PotreeConverter/src/main.cpp

```diff
@@ -179,7 +180,26 @@ PotreeArguments parseArguments(int argc, char **argv){
 
 		a.aabbValues = aabbValues;
 	}
-
+	if (args.has("metadata_processing")) {
+		vector<double> aabbValues;
+        auto vtmMetadata = parseVTMmetadata(a.metadataProcessingFile);
+		double paddingVTM = 300;
```

Comment:
define `paddingVTM` on the top of the file, like so `constexpr double PADDING_VTM = 300;`